backend/uclapi/resources/views.py

```diff
@@ -0,0 +1,45 @@
+import os
+import requests
+import xml.etree.ElementTree
+
+from rest_framework.decorators import api_view
+
+from roombookings.decorators import does_token_exist
+
+from roombookings.helpers import PrettyJsonResponse as JsonResponse
```

Comment:
This is now `from common.helpers import PrettyJsonResponse as JsonResponse`. The JsonResponse objects takes an extra parameter: `rate_limiting_data=kwargs` to handle rate limit headers.

Reply:
changed